src/language/languageServerController.ts

```diff
@@ -138,7 +138,7 @@ export default class LanguageServerController {
     }
 
     // Stop the language server
-    this._client.stop();
+    void this._client.stop();
```

Comment:
what happens if this throws an error?

Reply:
This is the same as `this._client.start()`, the error is being caught by the extension and shows the modal with the error that the language server can not start/stop.

Reply:
Ideally I think we'd have a comment here explaining that reasoning.